### PR TITLE
Added babel preset & npm start command

### PR DESCRIPTION
**Pull request by gerhardsletten**
So it is easier to try the example..

### CODE REVIEW

File: examples/hello/package.json [hunk 1]
```diff
@@ -3,8 +3,14 @@
...
   "version": "0.0.1",
   "private": true,
+  "scripts": {
+    "start": "babel-node --stage es2015,stage-2 server.js",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
   "dependencies": {
     "babel": "^5.8.3",
+    "babel-preset-es2015": "^6.5.0",
+    "babel-preset-stage-2": "^6.5.0",
```
Comment (revivek) [inline examples/hello/package.json]: Is this actually necessary? I don't think we use any syntax from stage 2 or 3.

https://babeljs.io/docs/plugins/preset-stage-2/

File: examples/hello/package.json [hunk 2]
```diff
@@ -3,8 +3,14 @@
   "description": "An example app demonstrating how Oy might be used.",
   "version": "0.0.1",
   "private": true,
+  "scripts": {
+    "start": "babel-node --stage es2015,stage-2 server.js",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
   "dependencies": {
     "babel": "^5.8.3",
+    "babel-preset-es2015": "^6.5.0",
```
Comment (revivek) [inline examples/hello/package.json]: I believe presets were introduced in `babel` 6.0.0 and up, and the version installed is 5.8.3.

**Comment by revivek** [comment]
Thanks for the PR! I know that `npm start` is a node convention but as far as I can tell, there is no benefit in the babel upgrade right now. If you'd like, feel free to move `babel-node server.js` into the `start` command. Also, there's a typo in the commit message with "present".

**Comment by gerhardsletten** [comment]
yes thats true, babel 5 will load it for you. Then this pull request is pointless, is it me that has to remove it?

**Comment by revivek** [comment]
I'm still open to moving `babel-node server.js` into `start`, so feel free to do so. Alternatively, you should be able to close the PR with the "Close pull request" button, but if that's not there, let me know and I'll close it out.

**Comment by gerhardsletten** [comment]
Okay, then I have updatet the start command and removed babel 6 presets

**Comment by revivek** [comment]
LGTM, thanks!

**Pull request merged**